server/grafana/dashboards/CITANodeInfoDashboard.json

```diff
@@ -1805,23 +1805,33 @@
       "strokeWidth": "1",
       "targets": [
         {
-          "expr": "(Node_Get_DirInfo_TotalFileSize{job=\"$job\",instance=\"$node:$port\"} * 1024)",
+          "expr": "Node_Get_DirInfo_TotalFileSize{job=\"$job\",instance=\"$node:$port\"} * 1024",
           "format": "time_series",
           "hide": false,
           "interval": "",
           "intervalFactor": 1,
-          "legendFormat": "use",
+          "legendFormat": "Node Dir Size by [ node : $node_id ]",
```

Comment:
“Node Dir Size by” should be "Node Dir Size Used by"

Reply:
done